# GPS home position in WGS84 coordinates.
uint64 timestamp			# Timestamp (microseconds since system boot)

float64 lat				# Latitude in degrees
float64 lon				# Longitude in degrees
float32 alt				# Altitude in meters (AMSL)

float32 x				# X coordinate in meters
float32 y				# Y coordinate in meters
float32 z				# Z coordinate in meters
